features/largest-contentful-paint.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
-name: Largest Contentful Paint
-description: TODO
+name: Largest contentful paint (LCP)
+description: The `LargestContentfulPaint` interface provides timing information about the largest image or text paint before user input on a web page. The key moment this API provides is known as the LCP metric.
```

For symmetry with other features like this:

```suggestion
description: The `largest-contentful-paint` performance entry and the `LargestContentfulPaint` API measures  the time it takes for the largest image or text to appear. Largest contentful paint (LCP) is a common metric for perceived loading times.
```